### PR TITLE
Feature backend wallet

### PR DESCRIPTION
**Pull request by Hordd-Scratch**

### CODE REVIEW

File: iconic/UserProfile/urls.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.urls import include, path
+
+from . import views
+
+urlpatterns = [
+    path('user/wallet/', views.UserWalletView.as_view(),
+         name='amount-wallet')
+]
```
Comment (Hrustyaschiy) [inline iconic/UserProfile/urls.py]: change path to "wallet/" and add "user/" path to iconic/urls.py like 
    "path('users/', include('UserProfile.urls'))"

File: iconic/UserProfile/views.py [hunk 2]
```diff
@@ -1,4 +1,19 @@
 from django.shortcuts import render
 
-# Create your views here.
+from rest_framework import permissions
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from iconic.UserProfile.models import Wallet
```
Comment (Hrustyaschiy) [inline iconic/UserProfile/views.py]: Wrong import, iconic.UserProfile.models => .models

File: iconic/UserProfile/serializers.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from iconic.UserProfile.models import Wallet
```
Comment (Hrustyaschiy) [inline iconic/UserProfile/serializers.py]: Wrong import iconic.UserProfile.models => .models

File: iconic/UserProfile/serializers.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from iconic.UserProfile.models import Wallet
+
+
+# This is custom wallet serializer
+class CustomWalletSerializer(serializers.ModelSerializer):
+    user_id = serializers.OneToOneField(required=True)
```
Comment (Hrustyaschiy) [inline iconic/UserProfile/serializers.py]: There is no OneToOneField in serializers, 
try user_id = serializers.PrimaryKeyRelatedField(read_only=True)

**Review by BarBoley — APPROVED** [review]

**Review by Hrustyaschiy — CHANGES_REQUESTED** [review]

**Review by BarBoley — APPROVED** [review]

**Review by BarBoley — APPROVED** [review]

File: iconic/accounts/urls.py [hunk 5]
```diff
@@ -6,4 +6,5 @@
     path('data/', views.UserRetrieveUpdateDestroyAPIView.as_view(),
          name='user-data'),
     path('users/', include(routers.router.urls)),
+    path('users/', include('UserProfile.urls'))
```
Comment (Hrustyaschiy) [inline iconic/accounts/urls.py]: Not there, this line should be in iconic/iconic/urls.py

**Review by Hrustyaschiy — CHANGES_REQUESTED** [review]

File: iconic/accounts/urls.py (hunk 5 above)
Comment (Hordd-Scratch) [reply iconic/accounts/urls.py]: you make me sad

**Review by Hrustyaschiy — APPROVED** [review]

**Pull request merged**